packages/cubejs-api-gateway/src/helpers/sqlRunnerHelper.ts

```diff
@@ -0,0 +1,17 @@
+import { UserError } from '../UserError';
+
+// Determine SQL query type and add LIMIT clause if needed
+export const shouldAddLimit = (sql: string): Boolean => {
```

Comment:
Could you please rename this file (as well as appropriate test file) to `shouldAddLimit`?